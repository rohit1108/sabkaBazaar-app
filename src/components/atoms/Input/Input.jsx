import "./Input.scss";
const Input = ({type,id, className, onChange, values}) =>{
        return (
            <input
                type={type}
                id={id}
                className={className}  
                value={values}
                onChange={onChange}
            />
        )
     }
export default Input