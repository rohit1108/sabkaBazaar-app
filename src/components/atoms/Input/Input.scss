input{
    padding: 0.35em 0;
    display: block;
    width: 90%;
    border: none;
    border-bottom: 2px solid rgb(173,168,165);
    outline: 0;
    
    margin-bottom: 1.5em;
    // &:hover{
    //     border: none;
    //     border-bottom: 1px solid rgb(173,168,165);
    // }
    &:focus{
        border: none;
        border-bottom: 2px solid rgb(53, 202, 239);
        // padding: 1em 0;
        
    }

}