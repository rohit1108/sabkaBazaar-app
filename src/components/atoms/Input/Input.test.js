import './Input';
//import { getByRole, screen} from '@testing-library/react';
import Input from './Input';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
//import { getAllJSDocTags } from 'typescript';

test('should have right Input comp snapshot', () => {
    const tree = renderer.create(<Input/>).toJSON();
    expect(tree).toMatchSnapshot();
    
  });

  it('renders the comp properly', () => {
    const divEl = document.createElement('div');
    ReactDOM.render(<Input />, divEl);
  });

