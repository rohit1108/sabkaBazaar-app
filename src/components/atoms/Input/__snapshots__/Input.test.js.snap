// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render email input 1`] = `<input />`;

exports[`render input 1`] = `<input />`;

exports[`should have right Button comp snapshot 1`] = `<input />`;

exports[`should have right Input comp snapshot 1`] = `<input />`;
